lib/babe/build.go

```diff
@@ -246,3 +248,63 @@ func extrinsicsToBody(txs []*transaction.ValidTransaction) (*types.Body, error)
 
 	return types.NewBodyFromExtrinsics(extrinsics)
 }
+
+func determineError(res []byte) string {
```

Comment:
I'm just building an error text string to include the error type in error messages.  Do you think I should create a struct for these errors to include error type, error description text, and module/error codes for dispatch errors?

Reply:
seems fine to me, I'm good with the error message + code